homeassistant/components/feedreader.py

```diff
@@ -0,0 +1,108 @@
+"""RSS/Atom feed reader for Home Assistant."""
+from datetime import datetime
+from logging import getLogger
+from time import sleep
+import threading
+from homeassistant.const import EVENT_HOMEASSISTANT_STOP
+from homeassistant.helpers import validate_config
+
+REQUIREMENTS = ['feedparser==5.2.1']
+
+_LOGGER = getLogger(__name__)
+
+DOMAIN = "feedreader"
+EVENT_FEEDREADER = "feedreader"
+POLLING_FREQUENCY = 3600  # In seconds
+
+
+class FeedManager(threading.Thread):
```

Comment:
Do not use threads, instead use an event listener.

This will fire each time at the hour:

```python
from homeassistant.helpers.event import track_utc_time_change

track_utc_time_change(hass, lambda now: update_feeds(), minute=0, second=0)
```